backend/src/crd/controller/scheduledworkflow/util/cron_schedule_test.go

```diff
@@ -108,7 +113,36 @@ func TestCronSchedule_GetNextScheduledEpoch(t *testing.T) {
 		Cron:    "0 * * * * * ",
 	})
 	assert.Equal(t, int64(10*hour+15*minute+minute),
-		schedule.GetNextScheduledEpoch(nil, defaultStartEpoch))
+		schedule.GetNextScheduledEpoch(nil, defaultStartTime, location))
+}
+
+func TestCronSchedule_GetNextScheduledEpoch_LocationsEnvSet(t *testing.T) {
+	locationString := "Asia/Shanghai"
+	viper.Set(TimeZone, locationString)
+	defer viper.Set(TimeZone, "")
+
+	location, err := time.LoadLocation(locationString)
+	assert.Nil(t, err)
+	startTime, err := time.Parse(time.RFC1123Z, "Mon, 11 Jan 2010 10:00:00 +0800")
+	assert.Nil(t, err)
+	endTime, err := time.Parse(time.RFC1123Z, "Mon, 11 Jan 2010 11:00:00 +0800")
+	assert.Nil(t, err)
+	lastJob, err := time.Parse(time.RFC1123Z, "Mon, 11 Jan 2010 10:20:00 +0800")
+	assert.Nil(t, err)
+	defaultStartTime, err := time.Parse(time.RFC1123Z, "Mon, 11 Jan 2010 10:15:00 +0800")
+	schedule := NewCronSchedule(&swfapi.CronSchedule{
+		StartTime: commonutil.Metav1TimePointer(v1.NewTime(startTime)),
+		EndTime:   commonutil.Metav1TimePointer(v1.NewTime(endTime)),
+		Cron:      "0 * * * * * ",
```

Comment:
Can you add a comment explain what this from does?

Reply:
Updated to use a const since we have this on 7 places through out the cron tests.